touchforms/formplayer/api.py

```diff
@@ -279,24 +280,37 @@ def server_down(cls):
 
 
 def post_data_helper(d, auth, content_type, url, log=False):
-    d['nav_mode'] = 'prompt'
     data = json.dumps(d)
     up = urlparse(url)
-    if log:
-        logging.info("Request to url: %s" % up.geturl())
     headers = {}
     headers["content-type"] = content_type
     headers["content-length"] = len(data)
-    headers["Cookie"] = 'sessionid=%s' % settings.FORMPLAYER_INTERNAL_AUTH_KEY
     conn = httplib.HTTPConnection(up.netloc)
     conn.request('POST', up.path, data, headers)
     resp = conn.getresponse()
-    if log:
-        logging.info("Response: %s" % resp)
     results = resp.read()
-    if log:
-        logging.info("Results: %s" % results)
     return results
+
+
+def formplayer_post_data_helper(d, auth, content_type, url):
+    d['nav_mode'] = 'prompt'
+    data = json.dumps(d)
+    up = urlparse(url)
+    logging.info("Request to url: %s" % up.geturl())
+    headers = {}
+    headers["content-type"] = content_type
+    headers["Content-Type"] = content_type
```

Comment:
why is this here twice?

Reply:
Copied over from the `validate_forms()` but I now read this is case-insensitive